lib/mapper.js

```diff
@@ -52,6 +49,7 @@ exports.page = function(page, settings){
 
 exports.screen = function(screen, settings){
   var result = createCommonGAForm(screen, settings);
```

Comment:
@ndhoule Good call. It now removes hostname (dh) and path (dp) if present on screen calls since those don't have meaning on screen calls.